imports/ui/signup/signup.actions.test.js

```diff
@@ -45,26 +40,25 @@ if (Meteor.isClient) {
           expect(Accounts.createUser).to.have.been.calledWithMatch({
             email: info.emailAddress,
             password: info.password,
+            tac: {
```

Comment:
should be removed along with the above mentioned code. You should however test for the existence of "tac"  by modifying the existing tests on **on-create-user.test.js**